activiti-cloud-services-org/activiti-cloud-services-org-api/src/main/java/org/activiti/cloud/organization/api/ModelContentConverter.java

```diff
@@ -41,17 +41,5 @@
      * @param modelContent the model content instance to convert
      * @return the bytes array
      */
-    byte[] convertToBytes(T modelContent);
-
-    /**
-     * Convert a bytes array to the handled model content entity, set the given id, and convert back to bytes array.
-     * If the id is already the expected one, it returns the given bytes unchanged.
-     * @param bytes the bytes to convert and fix
-     * @param modelContentId the correct model content id to set
-     * @return the fixed bytes array
-     */
-    default byte[] convertAndFixModelContentId(byte[] bytes,
-                                               String modelContentId) {
-        return bytes;
-    }
+    byte[] convertToBytes(ModelContent modelContent);
```

Comment:
Moving from T to ModelContent destroys the generics mechanism. This is necessary for strong control of types and allow injecting enterprise logic into community core (like,  for example, the forms related logic, which is not part of the community).